themes/default/content/product/_index.md

```diff
@@ -12,99 +12,116 @@ subheading: |
     and tame cloud complexity -- something we call Cloud Engineering.
 
 overview:
-    title: Unite developers, infrastructure teams, and security engineers
+    title: Universal Infrastructure as Code for Engineers
     description: |
-        By using the industry's best languages, you can skip learning proprietary, limited domain-specific
-        languages, and enable your team to work more productively and better together.
+        Ship infrastructure faster with software engineering and an API-first approach. Use Pulumi’s open source SDK with your favorite languages and software ecosystems to deploy on any cloud, and scale infrastructure as code across your team securely with enterprise-grade SaaS.
 
 key_features:
     title: Key features
     items:
-        - description: Build [Infrastructure as Code](/what-is/what-is-infrastructure-as-code/) in familiar languages
+        - title: "Author in any language, deploy to any cloud"
+          sub_title: "Pulumi Infrastructure as Code Engine"
+          description: 
+            Define infrastructure as code (IaC) in TypeScript/JavaScript, Python, Go, C#, Java, and YAML using your IDE and test frameworks for a fast inner dev loop. Deploy to [100+ providers](/registry/) like AWS, Azure, Google Cloud, and Kubernetes.
```

Comment:
global comment: these text blocks appear quite wide on my screen and might benefit from being compressed a little